ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/command/execute/ExecuteCommandActionManager.java

```diff
@@ -196,7 +196,7 @@ private void removeAction(CommandImpl command) {
       }
 
       // remove action group if it's empty
```

Comment:
If this comment isn't actual anymore let's remove it.

Reply:
Right, thanks, removed